03-rnaseq/differential-expression_rnaseq_01.Rmd

```diff
@@ -11,7 +11,13 @@ output:
 
 # Purpose of this analysis
 
-This notebook takes RNA-seq data and metadata from refine.bio and identifies differentially expressed genes between experimental groups. 
+This notebook takes RNA-seq expression data and metadata from refine.bio and identifies differentially expressed genes between two experimental groups. 
+
+Differential expression analysis is popular technique for expression data that will tell you what genes are most much higher or much lower between your experimental groups. 
```

Comment:
```suggestion
Differential expression analysis identifies genes with significantly varying expression among experimental groups. 
```

Reply:
I was trying to make the explanation as simple as possible but I think I overshot it.